docs/application/toc_all.md

```diff
@@ -129,8 +129,8 @@
 ### Media and Camera
 #### [Overview](/application/dotnet/guides/multimedia/overview.md)
 #### [Media Content and Metadata](/application/dotnet/guides/multimedia/media-content-metadata.md)
```

Comment:
When a node has children, the node must not have a link.

At this moment, when I click **Media Content and Metadata**  on https://docs1.stg.tizen.org/staging/1200/application/native/guides/multimedia/overview/,  the page loses its path on LNB.

If you want to keep the page, you have to the toc as follow:
```
#### Media Content and Metadata
##### [Overview](/application/dotnet/guides/multimedia/media-content-metadata.md)
##### [Media Content](/application/dotnet/guides/multimedia/media-content.md)
...
```

By the way, I think deleting this line is not bad. To do this, you have to change the overview.md file as well.
These files will be shown as follow:

`toc.all.md`
```
#### [Overview](/application/dotnet/guides/multimedia/overview.md)
#### [Media Content](/application/dotnet/guides/multimedia/media-content.md)
#### [Metadata](/application/dotnet/guides/multimedia/metadata.md)
```

`overview.md`
````
# Multimedia

Multimedia features introduce how you can handle multimedia resources, such as media files, images, audio, video, camera, and radio.

You can use the following multimedia features in your native applications:

- [Media Content](media-content.md)

  You can update database details due to file (or folder) creation or deletion. If a received file (or folder) does not exist in the file system, it is removed from the database. You can also retrieve a list of media folders, retrieve a list of media items, and monitor changes in the media database. You can search for specific media folders and read their information, and retrieve media folder content.

- [Metadata](metadata.md)

    You can manage audio file attributes, extract metadata from media files, and retrieve various information related to the MIME type.
 
- [Image Editing](image-edit.md)
  ...

```

Reply:
Fixed not to use media-content-metadata.md.